requirements.txt

```diff
@@ -1,13 +1,14 @@
-cert-core>=2.1.10
-cert-schema>=3.0.0a9
-chainpoint>=0.0.2
+cert-core>=3.0.0b1
+cert-schema>=3.0.0b4
```

Comment:
publish v3 and update here

Reply:
@lemoustachiste to update the dependencies in cert-tools https://github.com/blockchain-certificates/cert-tools/blob/master/requirements.txt#L2